.github/workflows/lint.yml

```diff
@@ -0,0 +1,19 @@
+name: Check Style and Quality
+
+on: [pull_request, push]
+
+jobs:
+    lint:
+        name: Lint
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - uses: actions/setup-python@v3
```

Comment:
Can we add a comment here mentioning why python is needed?

Reply:
It originates from the suggested usage here: https://github.com/pre-commit/action?tab=readme-ov-file#using-this-action
I can open a PR adding this comment if you want

Reply:
Thanks! I added something in https://github.com/vscode-neovim/vscode-neovim/pull/1865